instrumentation/ktor/ktor-2-common/library/src/main/kotlin/io/opentelemetry/instrumentation/ktor/v2_0/common/KtorHttpClientAttributesGetter.kt

```diff
@@ -34,4 +35,7 @@ internal object KtorHttpClientAttributesGetter : HttpClientAttributesGetter<Http
   override fun getServerAddress(request: HttpRequestData) = request.url.host
 
   override fun getServerPort(request: HttpRequestData) = request.url.port
+
+  private val urlTemplateAttributeKey = AttributeKey<String>("URL_TEMPLATE")
+  override fun getUrlTemplate(request: HttpRequestData): String? = request.attributes.getOrNull(urlTemplateAttributeKey)
```

Comment:
I'd suggest you to remove the ktor changes until there is a version of ktor that would support this. Once there is a version where this works you could add it back along with a test.